cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -44,7 +44,10 @@ PYTHON_BIN?=$(PYTHON_DIR)/python.exe
 # We might work with other Python versions
 LOCAL_PYTHON_VERSION?=$(shell $(PYTHON_BIN) -c 'import distutils.sysconfig; print( distutils.sysconfig.get_python_version() )')
 PYTHON_DYNLIBDIR:=$(shell $(PYTHON_BIN) -c 'from distutils import sysconfig; print( sysconfig.get_config_var("DESTSHARED") )')
-PYTHON_LIBDIR=$(PYTHON_DIR)/libs
+PYTHON_LIBDIR:=$(shell $(PYTHON_BIN) -c 'from distutils import sysconfig; print( sysconfig.get_config_var("LIBDIR") )')
+ifeq ($(PYTHON_LIBDIR),None)
+    PYTHON_LIBDIR=$(PYTHON_DIR)/libs
+endif
```

Comment:
How is this change related to this PR? If it's not related please open a separate PR or at least put it into a separate commit with a descriptive commit message (so that this PR has two commits).